patterns/common/sailfish-device-capabilities.yaml.template

```diff
@@ -0,0 +1,8 @@
+Description: All available capabilities. (Template file lists all, final .yaml will reflect only what device needs)
+Name: sailfish-device-capabilities
+Requires:
+%have_vibrator- pattern:sailfish-ha-vibrator
```

Comment:
What i meant is that capabilities are really device configuration features and could be just added to the existing jolla-configuration-@DEVICE@.yaml in case the configuration enables those. I don't currently understand the need for the sailfish-device-capabilities.yaml as it is anyway device specific in the end.